pyttsx3/drivers/espeak.py

```diff
@@ -169,50 +178,53 @@ def _onSynth(self, wav, numsamples, events):
                 )
 
             elif event.type == _espeak.EVENT_MSG_TERMINATED:
-                # Final event indicating synthesis completion
+                # Handle utterance completion
                 if self._save_file:
+                    # Save audio to file if requested
                     try:
                         with wave.open(self._save_file, "wb") as f:
-                            f.setnchannels(1)  # Mono
-                            f.setsampwidth(2)  # 16-bit samples
-                            f.setframerate(22050)  # 22,050 Hz sample rate
+                            f.setnchannels(1)
+                            f.setsampwidth(2)
+                            f.setframerate(22050)
                             f.writeframes(self._data_buffer)
-                        print(f"Audio saved to {self._save_file}")
+                        logging.debug(f"Audio saved to {self._save_file}")
                     except Exception as e:
                         raise RuntimeError(f"Error saving WAV file: {e}")
                 else:
+                    # Playback temporary file (if not saving to file)
                     try:
                         with NamedTemporaryFile(
                             suffix=".wav", delete=False
                         ) as temp_wav:
                             with wave.open(temp_wav, "wb") as f:
-                                f.setnchannels(1)  # Mono
-                                f.setsampwidth(2)  # 16-bit samples
-                                f.setframerate(22050)  # 22,050 Hz sample rate
+                                f.setnchannels(1)
+                                f.setsampwidth(2)
+                                f.setframerate(22050)
```

Comment:
Why remove these comments?

Reply:
I have a feeling this might have been my formatter

Reply:
I lied.. i have no idea why i removed that